packages/client/src/rpc/modules/debug.ts

```diff
@@ -278,4 +288,80 @@ export class Debug {
       message: err.message.toString(),
     }
   }
+
+  /**
+   * Returns a limited set of storage keys belonging to an account.
+   * @param params An array of 5 parameters:
+   *    1. The hash of the block that contains the requested account storage.
```

Comment:
```suggestion
   *    1. The hash of the block at which to get storage from the state.
```
just to make it less ambiguous